package.json

```diff
@@ -35,7 +35,7 @@
 		{
 			"gzip": false,
 			"running": false,
-			"limit": "33 KB",
+			"limit": "34.1 KB",
```

Comment:
Here we have a little bit of problem. We need to find some creative solution in the written JS to make sure we get smallest size.

You can check with AI some variations of the code that can produce a smaller code when minified.

Reply:
@Soare-Robert-Daniel I have updated the code to reduce the file size with the latest commit. Please check it and let me know if any changes are needed.